### ISSUE TITLE
Support custom content

### ISSUE THREAD

**Issue by tumidi** [innodoc/innoconv#32]
- Custom content is placed in the folder structure next to the actual course content (e.g. `de/_content/about.md`).
- It can have and reference static files just like any other content (localized `de/_content/_static` or non-localized `_content/_static`).

###### YAML (content source)

```yaml
custom_content:
- name: about
  title_de: Über den Kurs
  title_en: About the course
  link_in_nav: true
  link_in_footer: true
- name: license
  title_de: Lizenz
  title_en: License
  place_in_footer: true
```

If there is neither `link_in_nav`, `link_in_footer` nor `place_in_footer` present the compiler should emit a warning and not copy the custom content.

##### See also

- innodoc/innodoc-webapp#44

**Comment by tumidi** [comment]
mentioned in issue innodoc-webapp#44

**Comment by tumidi** [comment]
In GitLab by @ramsesoriginal on Feb 11, 2019, 16:26

mentioned in commit 6fd4e5a69f0bc4e8b515a9dde8a8d9083ca73a53

**Comment by tumidi** [comment]
closed via commit 634526ea8df27d2463119c151729bf68b8b8ff7f

**Comment by tumidi** [comment]
mentioned in commit 634526ea8df27d2463119c151729bf68b8b8ff7f
